vault/logical_system_helpers.go

```diff
@@ -67,6 +67,12 @@ var (
 		}
 	}
 
+	pathLicenseReadSigned = func(b *SystemBackend) framework.OperationFunc {
```

Comment:
Does this need to be in OSS?  It doesn't look like anything is calling it.

Reply:
I was just following the convention with `pathLicenseRead` and `pathLicenseUpdate` above. I don't imagine anything in OSS is calling those either?

Reply:
Ok, at some point it may be worth cleaning this up.  It doesn't hurt, but it's also unnecessary.

Reply:
Mostly I'm just interested in consistency, as I feel like that makes everything easier to reason about. I'm happy to remove all 3 of them.

Reply:
That would be my (mild) preference.

Reply:
Done